website_partner_social_network/__openerp__.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
```

Comment:
same remarks as above: headers, description in README, remove empty keys, version number